src/utils/utils.ts

```diff
@@ -0,0 +1,32 @@
+import { LOAD_MOVIES_AMOUNT } from '../state/constants';
+import SearchParam, { OFFSET } from '../types/search-param';
+
+export function getUrlParams() {
```

Comment:
We discussed that values should be passed via function arguments. Let's not take them from `window.location.search`. It will give us more flexibility and better testability.

It can be something like
```
const getUrlParams = (options: UrlSearchParams) = > {
   const searchParams = new URLSearchParams(options);
   // and then your current code
}
```